frontend/public/locales/en/translation.json

```diff
@@ -2115,7 +2116,8 @@
     "submariner.install.form.gateways.labelHelp": "The number of worker nodes  that will be used to deploy the Submariner gateway component on the managed cluster. If the value is greater than 1, the Submariner gateway HA will be enabled automatically. (default 1)",
     "submariner.install.form.gateways.placeholder": "Enter gateway count",
     "submariner.install.form.instancetype": "Instance type",
-    "submariner.install.form.instancetype.labelHelp": "The Amazon Web Services EC2 instance type of the gateway node that will be created on the managed cluster. (default c5d.large)",
+    "submariner.install.form.instancetype.labelHelp.aws": "The Amazon Web Services EC2 instance type of the gateway node that will be created on the managed cluster. (default c5d.large)",
```

Comment:
Not sure how it is in other UI instances but typically we would have the parenthesis before the period. I changed it in an example above but just do whatever keeps it consistent across the UI.

Reply:
I made the period at the end for all Submariner strings